data/tlg0732/tlg011/tlg0732.tlg011.1st1K-grc1.xml

```diff
@@ -0,0 +1,4487 @@
+                </respStmt>
+                <respStmt>
+                    <persName>Thibault Clérice</persName>
+                    <resp>Lead Developer (University of Leipzig)</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg0732.tlg011.1st1K-grc1.xml</idno>
+                <availability>
+                    <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+                        License</p>
+                </availability>
+                <date>2017</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United State</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title>Supplementum Aristotelicum</title>
+                            <editor>
+                                <persName>
+                                    <name>Ivo Bruns</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg0732">Alexander of Aphrodisias</author>
+                            <imprint>
+                                <publisher>Reimer</publisher>
+                                <pubPlace>Berlin</pubPlace>
+                                <date>1887</date>
+                            </imprint>
+                            <biblScope unit="volume">2</biblScope>
+                        </monogr>
+                        <ref target="http://www.archive.org/stream/p1supplementumar02akaduoft#page/n123/mode/2ups">Internet Archive</ref>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
+            <refsDecl n="CTS">
+                <cRefPattern matchPattern="(.+)" n="section"
+                    replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"
+                />
+            </refsDecl>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage>
+                <language ident="grc">Greek</language>
+                <language ident="lat">Latin</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change/>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="grc" n="urn:cts:greekLit:tlg0732.tlg011.1st1K-grc1">
+
+                    <div type="textpart" subtype="section" n="praef">
```

Comment:
I think this work also needs another layer of citation units below "section". Some of the sections are too long to be displayed well. I would either number the `<p>` elements or enclose them with `<div>` elements.

Reply:
There are barely more paragraphs than sections (28 paragraphs and 24 sections), so I don't think that would be of much help. Maybe "page" instead of "paragraph"?

@sonofmun

Reply:
Hmm, I am reluctant to add in a citation level that is edition-dependent, like page numbers. But there are some really long paragraphs in this work. We could maybe do sentences: just enclose everything that ends with a "." in a new `<div>`. Because at this point, some of the citation units are so long as to basically be meaningless.

Reply:
That is going to take a while. Just "." or "·" and ";" as well?

@sonofmun

Reply:
This looks good. I hope you did it with a regular expression instead of doing it by hand!